extensions/reactive-streams-operators/smallrye-reactive-streams-operators/runtime/pom.xml

```diff
@@ -45,6 +45,10 @@
             <groupId>io.smallrye</groupId>
             <artifactId>smallrye-context-propagation-propagators-rxjava2</artifactId>
         </dependency>
+        <dependency>
```

Comment:
@aloubyansky, should this be needed taking https://github.com/quarkusio/quarkus/blob/8f99dd4bac379ba35f9d14d2393b0dd6d61e05e8/extensions/reactive-streams-operators/smallrye-reactive-streams-operators/deployment/pom.xml#L36-L39 into account?

@mkouba suggested to ask you

Reply:
In other words, if I have an deployment module of extension `Alpha` with dependency to a deployment module of extension `Bravo` then the runtime module of `Bravo` must declare the dependency on the runtime `Alpha` as well, otherwise we get `NoClassDefFoundError` for a template class from `Bravo`... and suppose the app only depends on `Alpha` runtime.

Reply:
@michalszynkiewicz ^ is that correct?

Reply:
yes, that is my understanding of the situation.

Reply:
Unless you had any specific reason for adding that dependency, no.

Reply:
@aloubyansky I'm not sure I understand your answer ;-)

Reply:
I thought you asked me whether the dependency is necessary.

Reply:
So yes, you still need to declare the runtime dependency in the runtime artifact. It's not added automatically.